task_2.py

```diff
@@ -0,0 +1,18 @@
+"""Introduction to programming’: Task 2,
+   Kyshynska Taisiia"""
+
+print('Introduction to programming’: Task 2')
+print('Kyshynska Taisiia')
+
+TEMPLATE = "Input {} = "
+while True:
+    try:
+        x = int(input(TEMPLATE.format('x')))
+        y = int(input(TEMPLATE.format('y')))
+        z = int(input(TEMPLATE.format('z')))
```

Comment:
for instance, your user enter **x** correctly, **y** too, but he make mistake in **z** why does he need to refill **x and y**